test/WebJobs.Script.Tests/Helpers/SimpleWebTokenTests.cs

```diff
@@ -56,6 +57,39 @@ public void CreateTokenShouldCreateAValidToken()
             Assert.Equal($"exp={timeStamp.Ticks}", decrypted);
         }
 
+        [Fact]
+        public void ValidateTokenUsesContainerEncryptionKeyIfAvailable()
+        {
+            var containerEncryptionKey = TestHelpers.GenerateKeyBytes();
+            var containerEncryptionStringKey = TestHelpers.GenerateKeyHexString(containerEncryptionKey);
+
+            var websiteAuthEncryptionKey = TestHelpers.GenerateKeyBytes();
+            var websiteAuthEncryptionStringKey = TestHelpers.GenerateKeyHexString(websiteAuthEncryptionKey);
+
+            var timeStamp = DateTime.UtcNow.AddHours(1);
+
+            Environment.SetEnvironmentVariable(EnvironmentSettingNames.ContainerEncryptionKey, containerEncryptionStringKey);
```

Comment:
Need to update the cleanup done in Dispose for your new setting, otherwise it will leak out and potentially affect other test classes

Reply:
Added cleanup for the new setting also.